docs/content/4.API/2.components.md

```diff
@@ -0,0 +1,36 @@
+# Components
+
+Components API for Nuxt i18n module.
+
+## `<NuxtLinkLocale>`
+This component is built on top of [`<NuxtLink>`](https://nuxt.com/docs/api/components/nuxt-link#nuxtlink) but changes the default behavior by internally using [`localePath()`](/api/vue#localepath) to make it easier to link to localized routes.
+
+### Examples
+
+#### Basic usage
+```vue
+<template>
+  <NuxtLinkLocale to="/">{{ $t('home') }}</NuxtLinkLocale>
+  <!-- equivalent to -->
+  <NuxtLink :to="localePath('/')">{{ $t('home') }}</NuxtLink>
```

Comment:
Strictly, `localePath` requires a call to `useLocalePath` with the script block.

I think it would be better to write the equivalent separately as follows, instead of writing it here.

```vue
<script setup>
const localePath = useLocalePath()
</script>

<template>
  <NuxtLink :to="localePath('/')">{{ $t('home') }}</NuxtLink>
</template>
```